sql/table_test.go

```diff
@@ -168,7 +175,9 @@ func TestMakeTableDescIndexes(t *testing.T) {
 		}
 		schema, err := makeTableDesc(stmt[0].(*parser.CreateTable))
 		if err != nil {
```

Comment:
I've left it as is and have added a new test case to the tests that catch the [0] index not being primary problem.